source/documentation/tools/upgrading.md

```diff
@@ -0,0 +1,38 @@
+# Upgrading RStudio
+
+When moving between RStudio releases there are some differences that require either user intervention or caution.
+
+## Why should I upgrade?
+
+Upgrading to the latest version of RStudio has a number of benefits including newer versions of R, improved packages management options.
+
+## 2.2.6 to 3.0.11
```

Comment:
should this be 3.0.12 now?

Reply:
Yeah, it'll probably be 3.0.26 by the time we've fixed the bug :D :D